requirements/extras.txt

```diff
@@ -5,5 +5,5 @@ lmfit
 mpmath
 numba
 numba-scipy
-pytest >= 5.1
```

Comment:
I tentatively removed pytest from `extras.txt` since it was not included in `options.extras_require` and since it is already included in the testing requirements.

Reply:
Weird.  That probably shouldn't have been there in the first place.